test/functional/tests/volumes/test_multilevel_cache_3.py

```diff
@@ -0,0 +1,75 @@
+#
+# Copyright(c) 2022 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+
+import pytest
+from storage_devices.disk import DiskType, DiskTypeSet, DiskTypeLowerThan
+from core.test_run import TestRun
+from api.cas.cache import CacheMode, casadm
+from test_utils.size import Size, Unit
+from test_tools.disk_utils import Filesystem
+from test_tools import fs_utils
+from .test_raid_as_cache import create_files_with_md5sums, compare_md5sums
+
+mount_point = "/mnt/test"
+
+
+@pytest.mark.require_disk("cache_1", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("cache_2", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core_1", DiskTypeLowerThan("cache_1"))
+@pytest.mark.require_disk("hdd", DiskTypeSet([DiskType.hdd]))
+def test_multilevel_cache_3():
+    """
+    title:
+      Test multilevel cache triple.
+    description:
+      CAS Linux is able to use 3-level cache in different cache mode and remove it gracefully.
+    pass_criteria:
+      - Succesfully created 3-level cache.
+      - Succesfully mounted filesystem on CAS device.
+      - md5 sums are correct.
+    """
+    with TestRun.step("Prepare devices"):
+        core_dev_1 = TestRun.disks["core_1"]
+
+        cache_hdd = TestRun.disks["hdd"]
+        cache_hdd.create_partitions([Size(3.2, Unit.GibiByte)])
+        cache_hdd = cache_hdd.partitions[0]
+
+        cache_dev_1 = TestRun.disks["cache_1"]
+        cache_dev_1.create_partitions([Size(3.2, Unit.GibiByte)])
+        cache_dev_1 = cache_dev_1.partitions[0]
+
+        cache_dev_2 = TestRun.disks["cache_2"]
+        cache_dev_2.create_partitions([Size(3.2, Unit.GibiByte)])
+        cache_dev_2 = cache_dev_2.partitions[0]
+
+    with TestRun.step("create cache in WT mode and add core to it"):
+        cache_WT = casadm.start_cache(cache_dev=cache_dev_1, cache_mode=CacheMode.WT)
+        core_WT = cache_WT.add_core(core_dev=core_dev_1)
+
+    with TestRun.step("create cache in WB mode"):
+        cache_WB = casadm.start_cache(cache_dev=cache_dev_2, cache_mode=CacheMode.WB)
+
+    with TestRun.step("Add first CAS device as core to second"):
```

Comment:
You can describe it more directly as creating second layer cache by setting exported object as core.